hack/installers/install-codegen-go-tools.sh

```diff
@@ -41,9 +41,6 @@ go_mod_install k8s.io/code-generator/cmd/defaulter-gen
 go_mod_install k8s.io/code-generator/cmd/informer-gen
 go_mod_install k8s.io/code-generator/cmd/lister-gen
 
-# We still install openapi-gen from go.mod since upstream does not utilize release tags
-go_mod_install k8s.io/kube-openapi/cmd/openapi-gen
```

Comment:
I think this removal caused the codegen error. Has this binary been moved/renamed in 1.29.2?

Reply:
re-added, let's see how it behaves